include/revng/mlir/CMakeLists.txt

```diff
@@ -1 +1,17 @@
+# This policy mimics the behavior set in `CMakePolicy.cmake` in the official
```

Comment:
I don't believe we discussed this particular issue. Note that this is just moving existing code from `mlir/Dialect/Clift/IR/CMakeLists.txt` to `mlir/CMakeLists.txt`, since they're needed by many MLIR related `CMakeLists.txt`-files.

Reply:
:+1: